src/Query/UdpTransportExecutor.php

```diff
@@ -208,7 +209,13 @@ public function query(Query $query)
                 ));
                 return;
             }
-
+            if($response->rcode==Message::RCODE_NAME_ERROR){
```

Comment:
```suggestion
            if ($response->rcode == Message::RCODE_NAME_ERROR) {
```
Added some whitespace to match our used format. The same case occurs in a few other lines you've added.

Reply:
OK,you r right